[2024-09-21T12:22:53.352+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_saver_v01.insert_data scheduled__2024-09-21T00:00:00+00:00 [queued]>
[2024-09-21T12:23:03.743+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_saver_v01.insert_data scheduled__2024-09-21T00:00:00+00:00 [queued]>
[2024-09-21T12:23:03.786+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-21T12:23:03.888+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2024-09-21T12:23:03.896+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-21T12:23:11.771+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_data> on 2024-09-21 00:00:00+00:00
[2024-09-21T12:23:12.042+0000] {standard_task_runner.py:55} INFO - Started process 13522 to run task
[2024-09-21T12:23:12.550+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_saver_v01', 'insert_data', 'scheduled__2024-09-21T00:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/dag_saver.py', '--cfg-path', '/tmp/tmp0eb2r04t']
[2024-09-21T12:23:12.563+0000] {standard_task_runner.py:83} INFO - Job 99: Subtask insert_data
[2024-09-21T12:23:22.428+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_saver_v01.insert_data scheduled__2024-09-21T00:00:00+00:00 [running]> on host 16da3808dee2
